gourmet/gtk/recipe.py

```diff
@@ -0,0 +1,787 @@
+from gettext import gettext as _
+import os as _os
+
+from gi.repository import (
+    Gdk as _Gdk,
```

Comment:
What's with the `_`-prefixed imports and function definitions?

Reply:
[Per PEP 8](https://www.python.org/dev/peps/pep-0008/#id36), this signifies internal use and prevents the symbol from being imported when doing `from x import *`.